hikari/impl/rest.py

```diff
@@ -764,6 +764,22 @@ async def edit_channel(
         response = typing.cast(data_binding.JSONObject, raw_response)
         return self._entity_factory.deserialize_channel(response)
 
+    async def follow_channel(
+        self,
+        origin_channel: snowflakes.SnowflakeishOr[channels.GuildNewsChannel],
+        target_channel: snowflakes.SnowflakeishOr[channels.GuildChannel],
+        *,
+        reason: undefined.UndefinedOr[str] = undefined.UNDEFINED,
```

Comment:
Do we need to amend audit log code to support this?

Reply:
Nope, it's just a webhook create entry